files/grafana/prometheus.yml.j2

```
@@ -0,0 +1,12 @@
+global:
+  scrape_interval:     15s # By default, scrape targets every 15 seconds.
+  evaluation_interval: 15s # By default, scrape targets every 15 seconds.
+  # scrape_timeout is set to the global default (10s).
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['localhost:9090']
+#      - targets: [backup:9100]
+#        labels:
+#          group: 'Disks'
+# TODO add entries here
```

```suggestion
{% for host in groups["all"] %}
        - {{ hostvars[host].ansible_host }}:9100
{% endfor %}
```

This would just monitor the hardware stats of all hosts in your inventory.

---

hmm, should i exclude the domjudge things?

---

Probably better, not sure if we can filter in the groups['all'] to remove the `domjudge` machines (I'll look for that tomorrow) and we can also just set a firewall log to not expose the metrics to you